plugins/modules/packaging/os/homebrew_cask.py

```diff
@@ -454,18 +465,50 @@ def _current_cask_is_installed(self):
             self.message = 'Invalid cask: {0}.'.format(self.current_cask)
             raise HomebrewCaskException(self.message)
 
-        cmd = [
-            "{brew_path}".format(brew_path=self.brew_path),
-            "cask",
-            "list",
-            self.current_cask
-        ]
+        if self._brew_cask_command_is_deprecated():
+            base_opts = [
+                "{brew_path}".format(brew_path=self.brew_path),
+                "list",
+                "--cask",
+            ]
+        else:
+            base_opts = [
+                "{brew_path}".format(brew_path=self.brew_path),
+                "cask",
+                "list",
+            ]
+
+        cmd = base_opts + [self.current_cask]
         rc, out, err = self.module.run_command(cmd)
 
         if rc == 0:
             return True
         else:
             return False
+
+    def _get_brew_version(self):
+        if self.brew_version:
+            return self.brew_version
+
+        cmd = [self.brew_path, '--version']
+
+        process = subprocess.run(cmd, stdout=subprocess.PIPE)
```

Comment:
`in _get_brew_version\nAttributeError: 'module' object has no attribute 'run'\n",`

wasn't aware that `subprocess.run()` is only available in python3.5 <=